deploy/templates/nstemplatetiers/base/cluster.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: base-cluster-resources
+objects:
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-compute
+  spec:
+    quota:
+      hard:
+        limits.cpu: 20000m
+        limits.memory: 7Gi
+        limits.ephemeral-storage: 7Gi
+        requests.cpu: 1750m
+        requests.memory: 7Gi
+        requests.storage: 25Gi
```

Comment:
We need to set 25Gi for the existing `basic` tier only. Not for the new one.
Users provisioned to the new tier won't have workspaces in two namespaces.
But existing users in basic tier may have two namespaces with che workspaces.

And one more thing. I just realized that we actually have che test accounts in basicdeactivationdisabled tier too.
So, let's migrate this tier too:
- create a new tier `basedeactivationdisabled`
- set the storage quota to 25 in the existing `basicdeactivationdisabled`
- then we will migrate the users and kill the existing `basicdeactivationdisabled` as well as `basic`.

Reply:
Ah, that makes more sense. I'll make the changes.

Reply:
Done in a9ddde029fb06e08c35476a48b3a4f8d03976116